src/testsuite/vec_int32_dummy.c

```diff
@@ -23,6 +23,7 @@
 #include <pveclib/vec_int32_ppc.h>
 
 #ifdef _ARCH_PWR8
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
// clang does not support specific built-ins for new (PWR8) operations.
// These are supported for GCC.